snuba/configs/configuration.py

```diff
@@ -415,3 +428,16 @@ def delete_config_value(
             user=user,
             config_key=self._get_hash(),
         )
+
+    @classmethod
+    def config_key(cls) -> str:
+        return cls.__name__
```

Comment:
you said that `RegisteredClass` also has a `config_key` method but it represents a different thing. Maybe this function should be called something else that's more decriptive?